projects/wmf_default.yaml

```diff
@@ -171,9 +150,12 @@ paths:
                       fr.wikipedia.org:
                         - 'Utilisateur:ZéroBot/Log/Erreurs'
                         - 'Utilisateur:SyntaxTerror/Ajouts_du_modèle_Autorité'
+                        - '/^Utilisateur:[\s\S]+[Bb]rouillon'
```

Comment:
This encompasses all user pages that are drafts. But more generally, I think we could blacklist all user pages that have subpages, i.e. `/^Utilisteur:.+\/.+/`. Thoughts?